specs/core/0_beacon-chain.md

```diff
@@ -942,24 +942,25 @@ def get_crosslink_committees_at_slot(state: BeaconState,
     state_epoch_slot = state.slot - (state.slot % EPOCH_LENGTH)
     assert state_epoch_slot <= slot + EPOCH_LENGTH
     assert slot < state_epoch_slot + EPOCH_LENGTH
```

Comment:
I think this should be changed to:
```python
previous_epoch_start_slot = state.slot - state.slot % EPOCH_LENGTH - EPOCH_LENGTH
next_epoch_start_slot =  state.slot - state.slot % EPOCH_LENGTH + EPOCH_LENGTH
assert previous_epoch_start_slot <= slot < next_epoch_start_slot
```

Reply:
The logic is identical, right?
I do see it's also much more understandable, but also, the intention of `state_epoch_slot` is the avoid uint underflow: https://github.com/ethereum/eth2.0-specs/pull/429.

Reply:
I definitely prefer the readability...

Reply:
but yes, I believe it is the same functionality.

The underflow can be avoided using something like 
```python
current_epoch_start_slot = state.slot - state.slot % EPOCH_LENGTH
previous_epoch_start_slot = current_epoch_start_slot - EPOCH_LENGTH if state.slot >= EPOCH_LENGTH else current_epoch_start_slot
next_epoch_start_slot =  state.slot - state.slot % EPOCH_LENGTH + EPOCH_LENGTH
assert previous_epoch_start_slot <= slot < next_epoch_start_slot